modules/nodes-nodes-modifying-marking.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
```

Comment:
should be reduced to a single example command and included in "Working with nodes" in the evacuation section.